requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.11
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
because it was generated with 3.10 some dev dependencies were added, not super major, but might be good to regenerate it with 3.11

Reply:
Thanks for catching it. deps are regenerated